src/features/configTable/configTable.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from "react";
+import { Field, Form, FormRenderProps } from "react-final-form";
+import arrayMutators from "final-form-arrays";
+import { FieldArray } from "react-final-form-arrays";
+import {
+  Box,
+  Button,
+  Checkbox,
+  HStack,
+  FormLabel,
+  Input,
+  VStack,
+  Heading,
+  Spacer,
+  IconButton,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  FormErrorMessage,
+  FormControl,
+  FormHelperText,
+} from "@chakra-ui/react";
+import { FaPlus, FaTrash } from "react-icons/fa";
+
+type ValidationErrorType = {
+  configData: Employee[];
+};
+
+export type Employee = {
+  name: string;
+  surName: string;
+  employed: boolean;
+};
+
+const onSubmit = async (values: { configData: Employee[] }) => {
+  window.alert(JSON.stringify(values));
+};
+
+const validateForm = (values: { configData: Employee[] }) => {
```

Comment:
did you evaluated a Validation library like Zod or Yup?

Reply:
I think there is a task for this specifically but is not in the scope of this one.